### ISSUE TITLE
WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img[width="1"])

### ISSUE THREAD

**Issue by luisquintanilla** [microsoft/accessibility-insights-action#1194]
#### Issue
`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[Welcome to CityLights! [Inaccessible Home Page]](https://www.w3.org/WAI/demos/bad/before/home.html)
#### Element path
img[width="1"]
#### Snippet
`<img src="./img/marker2_t.gif" width="1" height="30px">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Microsoft Edge version 101.0.1210.47
This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
